### DISCUSSION TITLE
Parameter the number of recent activities in calendar views

### DISCUSSION THREAD

**Discussion by cariforef-normandie** [kimai/kimai#2556] [General]
Hi,

First of all, thank you for al your work, I really appreciate it !!
So, is it possible to have less or more than 10 recent activities for drag and drog in calendar view ? I found how to do this in the code (CalendarController.php line 98) but I would be nice to have this in the system parameters.

Regards,

Yann

**Comment by kevinpapst** [comment]
Hi Yann, I already worked on it, but it has absolutely no importance for me...
You could help with it if you want: https://github.com/kevinpapst/kimai2/pull/2291
